cmake/flash/CMakeLists.txt

```diff
@@ -130,7 +130,7 @@ endif()
 
 zephyr_get(WEST_DIR)
 if(WEST_DIR)
-  set(WEST "PYTHONPATH=${WEST_DIR}/src" "${PYTHON_EXECUTABLE};${WEST_DIR}/src/west/app/main.py;--zephyr-base=${ZEPHYR_BASE} ")
+  set(WEST "PYTHONPATH=${WEST_DIR}/src${SEP}$ENV{PYTHONPATH}" "${PYTHON_EXECUTABLE};${WEST_DIR}/src/west/app/main.py;--zephyr-base=${ZEPHYR_BASE} ")
```

Comment:
`SEP` variable is set from `doc/CMakeLists.txt` and seems to be somewhat ill conceived, but that is a separate issue.

https://github.com/zephyrproject-rtos/zephyr/blob/9cf89603dfffd320d9b3d127e5cccc1d8c40b24e/doc/CMakeLists.txt#L55-L59